config/env/production.js

```diff
@@ -47,13 +54,14 @@ if (AWSCreds) {
 
 // If running in Cloud Foundry with a redis service
 if (redisCreds) {
-  module.exports.session = {
-    adapter: 'redis',
+  session = _.extend({}, session, {
+    adapter: 'socket.io-redis',
```

Comment:
you were totally right, I looked again at the diff and saw that I removed the non-socket redis session information. This should be correct now.